<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2024-09-29 Sun 23:31 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Spatial Transcriptomics</title>
<meta name="author" content="chameko" />
<meta name="generator" content="Org Mode" />
<link rel="stylesheet" type="text/css" href="imagine.css" />
</head>
<body>
<div id="content" class="content">
<h1 class="title">Spatial Transcriptomics</h1>
<div id="outline-container-org7953472" class="outline-2">
<h2 id="org7953472">What is it</h2>
<div class="outline-text-2" id="text-org7953472">
<p>
As most human cells share essentially the same DNA, cell function cannot be described using DNA and is instead described by the proteins that the cells produce.
</p>
</div>
<div id="outline-container-orge634210" class="outline-3">
<h3 id="orge634210">mRNA</h3>
<div class="outline-text-3" id="text-orge634210">
<p>
mRNA is a molecule that contains the instructions that direct a cell to make a protein. As there is no commercial method to quantify all the proteins that are used in a singular cell we instead take the description of the entire repertoire of mRNA from a single cell. This is called the <b>transcritptome</b> and the process is called single cell RNA sequencing.
</p>
</div>
</div>
<div id="outline-container-orgc14561e" class="outline-3">
<h3 id="orgc14561e">scRNA-seq</h3>
<div class="outline-text-3" id="text-orgc14561e">
<p>
Most single cell RNA sequencing methods require the cell to be released intact and viable from tissues which are then assessed
</p>
</div>
<div id="outline-container-org3c3ffb3" class="outline-4">
<h4 id="org3c3ffb3">Negatives</h4>
<div class="outline-text-4" id="text-org3c3ffb3">
<p>
As the cell has to be removed intact from the tissue this can make some cells very hard to get (such as neurons). It also removes the cell from its neighbours, which may hold helpful information for defining cell and tissue function.
</p>
</div>
</div>
</div>
<div id="outline-container-orgfd4af62" class="outline-3">
<h3 id="orgfd4af62">Spatial Transcriptomics</h3>
<div class="outline-text-3" id="text-orgfd4af62">
<p>
Spatial transcriptomics evolved from the need to study these cells in their tissue. As the spatial information of the cell is preserved by studying the intact tissue it is called <i>spatial</i> trascriptomics.
</p>

<p>
Spatial transcriptomics aims to count the number of transcipts of a gene at distinct spatial locations in a tissue.
</p>

<p>
There are currently two predominant techniques, imaging and sequencing where the main trade off between technologies is the efficiency of the technique i.e. the proportion of transcripts of interest that are successfully counted.
</p>
</div>
</div>
</div>
<div id="outline-container-org915dfe5" class="outline-2">
<h2 id="org915dfe5">Research</h2>
<div class="outline-text-2" id="text-org915dfe5">
<ul class="org-ul">
<li>Williams, C.G., Lee, H.J., Asatsuma, T. et al. An introduction to spatial transcriptomics for biomedical research. Genome Med 14, 68 (2022). <a href="https://doi.org/10.1186/s13073-022-01075-1">https://doi.org/10.1186/s13073-022-01075-1</a></li>
<li>Marx, V. Method of the Year: spatially resolved transcriptomics. Nat Methods 18, 9â€“14 (2021). <a href="https://doi.org/10.1038/s41592-020-01033-y">https://doi.org/10.1038/s41592-020-01033-y</a></li>
</ul>
</div>
</div>
</div>
<div id="postamble" class="status">
<p class="author">Author: chameko</p>
<p class="date">Created: 2024-09-29 Sun 23:31</p>
<p class="validation"><a href="https://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
