<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2024-09-29 Sun 23:33 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>Portfolio 3</title>
<meta name="author" content="chameko" />
<meta name="generator" content="Org Mode" />
<link rel="stylesheet" type="text/css" href="imagine.css" />
</head>
<body>
<div id="content" class="content">
<h1 class="title">Portfolio 3</h1>
<p>
While my research speed has improved the effectiveness has not. My typically research pipeline looks like this
</p>
<ol class="org-ol">
<li>Start research on something</li>
<li>Get stuck</li>
<li>Find good resource that leads to other good resources</li>
<li>Gain good understanding of subject</li>
<li>Forget :(</li>
<li>Use list of collected links to do research again when I need to write something using said research</li>
<li>profit ???</li>
</ol>

<p>
Obviously not ideal. This can primarily be attributed to my poor note taking. One of the reasons I find note taking so hard is because I get immersed in my research, going from resource to resource without really writing everything down. Given I also have to maintain a portfolio of my work, its an area that requires investment. Right now I'm exploring using the Zettelkasten Method because it allows me to take notes with less structure which better fits how I think and allows me to quickly add concepts when researching, which will hopefully increase my engagement with the task. It also fits well with emacs (org-roam) which is the tool I use for my organisation and other text-based stuff.
</p>

<p>
Through my research I found an interesting article/paper [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8000474/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8000474/</a>] that talks about the idea of using gene expression signatures for prognosis. This helped better define my motivation as spatial transcriptomics is a tool for finding gene expression. It also provides a potential place to extend my model to, adding an extra block that uses the gene expression results to predict a cancer prognosis. 
</p>
</div>
<div id="postamble" class="status">
<p class="author">Author: chameko</p>
<p class="date">Created: 2024-09-29 Sun 23:33</p>
<p class="validation"><a href="https://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
